lib/hammer_cli_foreman/defaults.rb

```diff
@@ -0,0 +1,27 @@
+require 'hammer_cli'
+module HammerCLIForeman
+
+  class Defaults < HammerCLI::Defaults
```

Comment:
I implemented it that way before we changed to register providers, now we don't really need it.
ill refactor it and update you.
Thanks.